railties/lib/rails/application/configuration.rb

```diff
@@ -278,25 +278,25 @@ def respond_to_missing?(symbol, *)
       end
 
       private
-        def credentials_available_for_current_env?
-          File.exist?("#{root}/config/credentials/#{Rails.env}.yml.enc")
-        end
-
         def default_credentials_content_path
           if credentials_available_for_current_env?
-            File.join(root, "config", "credentials", "#{Rails.env}.yml.enc")
+            root.join("config", "credentials", "#{Rails.env}.yml.enc")
```

Comment:
This was failing in one test for me as `root` was a string there (not sure if it was a valid usecase). I guess Travis will show it.

Reply:
Looks like calling `app.root=` auto converts to a Pathname: https://github.com/rails/rails/blob/7c89948c416fbc32b59e33a0ab454545b4f6fed7/railties/lib/rails/engine/configuration.rb#L74
And the method that initialises the configuration also gets a Pathname from `find_root`: https://github.com/rails/rails/blob/3631d7eee4bd034f2eefe1b9892d5fcd565579ac/railties/lib/rails/application.rb#L379

I think that test is off.